example/src/example_description_handler.erl

```diff
@@ -1,28 +1,17 @@
 -module(example_description_handler).
```

Comment:
This file was not only formatted but adapted to work with the `cowboy` version `trails` expects.